.gitignore

```diff
@@ -52,3 +52,6 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# IDE
+.idea/
```

Comment:
Personal staff should not part of this project

Reply:
Done